src/raftstore/store/store.rs

```diff
@@ -770,6 +770,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
     fn on_raft_message(&mut self, mut msg: RaftMessage) -> Result<()> {
         let region_id = msg.get_region_id();
         if !self.is_raft_msg_valid(&msg) {
+            self.raft_metrics.message.drop_invalid_msg += 1;
```

Comment:
any reason to drop the message? stale, store not match or others? 

I think we should more detailed metric tags.